project/Dependencies.scala

```diff
@@ -3,48 +3,48 @@ import sbt._
 object Dependencies {
 
   object V {
-    val bouncyCastle = "1.69"
+    val bouncyCastle = "1.70"
     val betterFiles = "3.9.1"
     val cats = "2.7.0"
     val catsEffect = "3.3.5"
     val catsRetry = "2.1.0" // not published for CE3 yet
-    val circe = "0.14.1"
-    val ciris = "2.1.1"
-    val comcast = "3.0.3"
+    val circe = "0.14.2"
+    val ciris = "2.3.2"
+    val comcast = "3.1.3"
     val decline = "2.2.0"
-    val derevo = "0.12.6"
+    val derevo = "0.13.0"
     val doobie = "1.0.0-RC1"
-    val droste = "0.8.0"
+    val droste = "0.9.0"
     val enumeratum = "1.7.0"
-    val h2 = "1.4.200"
+    val h2 = "2.1.212"
     val javaxCrypto = "1.0.1"
-    val flyway = "8.0.0"
+    val flyway = "8.5.12"
     val fs2 = "3.1.1"
```

Comment:
Can we update fs2 as well? `val fs2 = "3.2.7"`

Reply:
Can I update it in next PR? I'd like to do such PRs systematically and I would have to test the full flow again

Reply:
It works :+1: